### DISCUSSION TITLE
RTCGQ11LM Sends On and Off when detecting no movement in Home Assistant

### DISCUSSION THREAD

**Discussion by beloso** [Koenkk/zigbee2mqtt#9673] [Devices]
I am trying to use Aqara RTCGQ11LM to add motion control to some lights.
I am using ZigBee2MQTT / HomeAssistant / NodeRed

In NodeRed, I am using the entity created for my sensor in HomeAssistant.
Whenever my sensor changes value (off -> on || on -> off ), the HomeAssistant entity emits 2 values. The first value it emits is the "old" state of the sensor, and next, it sends the correct value.

For instance, there is no motion for a while in that area, and when motion happens, it sends the value "off" and then "on."

Is there a workaround for this?

**Comment by bartplessers** [comment]
Related to #9519 ?
